File: code_blocks/🔌 Integrations/integrations/clevertap_1.java
// Configure Purchases SDK
Purchases.configure(this, "public_sdk_key", "my_app_user_id");

// Configure CleverTap SDK
CleverTapAPI clevertapDefaultInstance = CleverTapAPI.getDefaultInstance(this);

String cleverTapId = cleverTapDefaultInstance.getCleverTapId();
if (cleverTapId != null) {
  Map<String, String> attributes = new HashMap<String, String>();
	attributes.put("$cleverTapId", cleverTapId);

	Purchases.getSharedInstance().setAttributes(attributes);
}

File: code_blocks/🔌 Integrations/integrations/clevertap_1.kt
// Configure Purchases SDK
Purchases.configure(this, "public_sdk_key", "my_app_user_id")

// Configure CleverTap SDK
cleverTapDefaultInstance = CleverTapAPI.getDefaultInstance(this)

val cleverTapId = cleverTapDefaultInstance?.cleverTapID?.also {
   Purchases.sharedInstance.setAttributes(mapOf("$cleverTapId" to it))
}